src/resources/product.rs

```diff
@@ -87,7 +87,7 @@ pub struct Product {
     ///
     /// Only applicable to products of `type=good`.
     #[serde(default)]
-    pub shippable: bool,
+    pub shippable: Option<bool>,
```

Comment:
This change needs to be handled in codegen if it wants to be persisted (since the file is automatically generated).

But it doesn't seem necessary (because of `#[serde(default)]`)?.

Do you want to back out this change and I'll just merge in the `Customer` variant for `Event`?

Reply:
> This change needs to be handled in codegen if it wants to be persisted (since the file is automatically generated).

Sorry, my bad, how do I go about doing that?

> But it doesn't seem necessary (because of #[serde(default)])?.

Serde default only works for missing fields, I'm receiving `"shippable": null` in my `product.created` event. The deserialization was failing saying that `null` is not a valid boolean.

If you have any other ideas on how to solve this with serde, I can test it out.

> Do you want to back out this change and I'll just merge in the Customer variant for Event?

I'll split it up into two pull requests, should've done that from the start.